docs/Useful-snippets/snippets.rst

```diff
@@ -0,0 +1,13 @@
+Useful Snippets
+===============
+
+A collection of useful code snippets. Here you can find code that allows you to get to get started on common tasks in Mesa.
+
+Models with Discrete Time
+-------------------------
+If you have `Multiple` type agents and one of them has time attribute you can still build a model that is run by discrete time. In this example, each step of the model, and the agents have a time attribute that is equal to the discrete time to run its own step.
```

Comment:
> the agents have a time attribute that is equal to the discrete time

On a first pass, I find this sub-sentence confusing. The "time attribute" is an int/float. The "discrete time" is implemented as a List[Int] / List[float].

Also, in general, ABM models already execute on discrete time steps. I'd say it is more of "in particular times" / "custom time schedule." Also, better if initialized with concrete examples.

Reply:
@rht agreed. 
@JessicaGarson, can you submit an update to this?